api/v1/server/restapi/cilium_api.go

```diff
@@ -401,6 +401,9 @@ func (o *CiliumAPI) HandlerFor(method, path string) (http.Handler, bool) {
 	if _, ok := o.handlers[um]; !ok {
 		return nil, false
 	}
+	if path == "/" {
+		path = ""
```

Comment:
all the files in the api/v1 directory were modified using the generate-api target. This is probably part of the template go-swagger uses https://github.com/go-swagger/go-swagger/blob/master/generator/templates/server/builder.gotmpl#L268-L270